src/lib/store.js

```diff
@@ -115,6 +115,8 @@ export const reducers = {
       }),
       SET_THEME: (state, { meta }) => ({
         ...state,
+        // flush pending themes on save so that saved themes don't trigger modal
+        pendingTheme: null,
```

Comment:
This should probably get handled by dispatching a `clearPendingTheme` action wherever necessary, rather than a side effect here